api4/system_test.go

```diff
@@ -527,3 +527,26 @@ func TestS3TestConnection(t *testing.T) {
 		t.Fatal("should return error ")
 	}
 }
+
+func TestSupportedTimezones(t *testing.T) {
+	th := Setup().InitBasic()
+	defer th.TearDown()
+	Client := th.Client
+
+	supportedTimezonesFromConfig := th.App.Config().SupportedTimezones
+	supportedTimezones, resp := Client.GetSupportedTimezone()
+
+	CheckNoError(t, resp)
+	for _, timezone := range supportedTimezones {
+		found := false
+		for _, configTimezone := range supportedTimezonesFromConfig {
```

Comment:
nit: This does indeed verify that the client returns only configurations that are supported, but doesn't verify that it returns all of them. Would it be simpler just to use `assert.Equal(t, supportedTimezonesFromConfig, supportedTimezones)`, given that they are going to already be in the same order?